README.md

```diff
@@ -1,14 +1,12 @@
 ## AWS Observability Accelerator
 
-[![pre-commit](https://github.com/aws-observability/aws-observability-accelerator/actions/workflows/pre-commit.yaml/badge.svg?branch=main)](https://github.com/aws-observability/aws-observability-accelerator/actions/workflows/pre-commit.yaml)
-
 Welcome to the AWS Observability Accelerator!
 
 The AWS Observability Accelerator help you set up observability for your AWS environments with AWS-managed observability services such as Amazon Managed Service for Prometheus, Amazon Managed Grafana, AWS Distro for OpenTelemetry (ADOT) and Amazon CloudWatch.
 
-This repository provide shared artifacts (documentation, dashboards, alerting rules) for the [Terraform](https://github.com/aws-observability/terraform-aws-observability-accelerator) project and serves as a GitOps source repository.
+This repository provide shared artifacts (documentation, dashboards, alerting rules) for the [Terraform](https://github.com/aws-observability/terraform-aws-observability-accelerator) project and [CDK](https://github.com/aws-observability/cdk-aws-observability-accelerator) project. This repository also serves as a GitOps source repository.
 
-Checkout the [project documentation](https://aws-observability.github.io/terraform-aws-observability-accelerator/) for more informations.
+Checkout the [project documentation for Terraform](https://aws-observability.github.io/terraform-aws-observability-accelerator/) and [project documentation for CDK](https://aws-observability.github.io/cdk-aws-observability-accelerator/) for more information.
```

Comment:
same comment for "project documentation"

Reply:
fixed